package.json

```diff
@@ -59,19 +85,22 @@
 			},
 			{
 				"command": "camelk.integrations.remove",
-				"title": "Remove Integration"
+				"title": "Remove Camel-K Integration"
+			},
+			{
+				"command": "camelk.integrations.log",
+				"title": "View log for Camel-K Integration"
+			},
+			{
+				"command": "camel.integrations.startproxy",
```

Comment:
I think there is a typo, would be best with "camelk.integrations.startproxy" instead of "camel.integrations.startproxy"